puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy

```diff
@@ -28,9 +28,6 @@ pipeline {
         stage('Install Test') {
             agent { label 'el' }
 
-            environment {
-                VAGRANT_DEFAULT_PROVIDER = 'rackspace'
```

Comment:
What handles this now?

Reply:
This is part of an experiment, but I'm hoping that the Vagrantfile in the homedir helps to set the precedence. I'll try this on a VM to see if it works.